recipes/openjph/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
+import os
+
+required_conan_version = ">=1.53.0"
+
+class OpenJPH(ConanFile):
+    name = "openjph"
+    description = "Open-source implementation of JPEG2000 Part-15 (or JPH or HTJ2K)"
+    license = "BSD-2-Clause"
+    homepage = "https://github.com/aous72/OpenJPH"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("ht-j2k", "jpeg2000", "jp2", "openjph", "image", "multimedia", "format", "graphics")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_tiff": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_tiff": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.with_tiff:
+            self.requires("libtiff/4.6.0")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 14)
```

Comment:
As per https://github.com/aous72/OpenJPH/blob/master/CMakeLists.txt#L95, this should be 11 for the library itself right?

Also, we should remove the hardcoding of `CMAKE_CXX_STANDARD` in the following line too

Reply:
Oh, I read over the comment to be honest. Just fixed it. Would it be good practice to make the tests also avaialble? To my understanding it isn't really normally as conan is for providing libraries, not directly develop them from CCI recipes or do I see that wrong?